api/lambdas/sqs-to-converter/index.js

```diff
@@ -191,9 +191,9 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
 function addExtensionToConversion(conversion, extension) {
   const fhirBundle = conversion.fhirResource;
   if (fhirBundle?.entry?.length) {
-    for (const resource of fhirBundle.entry) {
-      if (!resource.extension) resource.extension = [];
-      resource.extension.push(extension);
+    for (const bundleEntry of fhirBundle.entry) {
+      if (!bundleEntry.resource.extension) bundleEntry.resource.extension = [];
+      bundleEntry.resource.extension.push(extension);
```

Comment:
This is just renaming the variable, as we weren't deconstructing...

Reply:
sorry, a bad diff from GH led me to think the only changes on the code were the variable names

<img width="1270" alt="image" src="https://github.com/metriport/metriport/assets/2132564/732898e7-c708-443b-b8e0-3e13b31792e1">